.github/workflows/ci.yml

```diff
@@ -142,6 +142,8 @@ jobs:
 
       - name: download deps
         run: go mod download
+      - name: Setup QEMU
+        run: docker run --rm --privileged upstream.azurecr.io/binfmt:latest --install all
```

Comment:
TODO: Change this image ref

Reply:
this registry has a purge task on that will get rid of images older than 30 or 90 days old. So, this might break at some point.

Reply:
this task has an exclusion option so maybe we can add it there

Reply:
I plan to change this ref.

Reply:
Changed this.